Open Judge System/Data/OJS.Data.Models/ProblemSubmissionTypeExecutionDetails.cs

```diff
@@ -1,21 +1,26 @@
-﻿namespace OJS.Data.Models
-{
-    using System.ComponentModel.DataAnnotations.Schema;
-
-    [Table("ProblemSubmissionTypesSkeletons")]
-    public class ProblemSubmissionTypeSkeleton
-    {
-        public int ProblemId { get; set; }
-
-        public virtual Problem Problem { get; set; }
-        
-        public int SubmissionTypeId { get; set; }
-
-        public virtual SubmissionType SubmissionType { get; set; }
-
-        /// <summary>
-        /// Gets or sets a predefined skeleton for the task and strategy
-        /// </summary>
-        public byte[] SolutionSkeleton { get; set; }
-    }
+﻿namespace OJS.Data.Models
+{
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    [Table("ProblemSubmissionTypeExecutionDetails")]
```

Comment:
The table name should be ProblemSubmissionTypesSkeletons - because in other cases migration is required